File: ADT/graphs/adjacency_list.py
# adjacency matrix
def createGraph(V, edges):
    mat = [[0 for _ in range(V)] for _ in range(V)]

    for it in edges:
        u = it[0]
        v = it[1]
        mat[u][v] = 1

        mat[v][u] = 1 #undirected graph
    return mat

def createGraph(V , edges):
    mat = [[0 for _ in range(V)] for _ in range(V)]

    for it in edges:
        u = it[0]
        v = it[1]
        mat[u][v] = 1
    return mat
# Adj list
def createGraph(V, edges):
    adj = [[] for _ in range(V)]
    for it in edges:
        u = it[0]
        v = it[1]
        adj[u].append(v)
    return adj

def createGraph(V, edges):
    adj = [[] for _ in range(V)]
    for it in edges:
        u = it[0]
        v = it[1]
        adj[u].append(v)
    return adj


def createGraph(V,edges):
    adj = [[] for _ in range(V)]
    for it in edges:
        u = it[0]
        v = it[1]
        adj[u].append(v)
    return adj

File: ADT/graphs/adjacency_list.js
// adjacency matrix
function createGraph(V, edges){
    let mat = Array.from({length: V}, () => Array(V).fill(0))
    for(let it of edges){
        let u = it[0]
        let v = it[1]

        mat[u][v] = 1
        mat[v][u] = 1
    }
    return mat
}

function createGraph(V, edges){
    let mat = Array.from({length: V}, () => Array(V).fill(0))

    for(let it of edges){
        let u = it[0]
        let v = it[1]

        mat[u][v] = 1
    }
    return mat
}

function createGraph(V, edges){
    let adj = Array.from({length: V}, () => [])
    for(let it of edges){
        let u = it[0]
        let v = it[1]
        adj[u].push(v)
    }
    return adj
}

function createGraph(V, edges){
    let adj = Array.from({length: V}, () => [])
    for(let it of edges){
        let u = it[0]
        let v = it[1]
        adj[u].push(v)
    }
    return adj
}